binding/binding_test.go

```diff
@@ -193,7 +337,12 @@ func TestBinding_GetName(t *testing.T) {
 }
 
 func TestBinding_Marshal(t *testing.T) {
```

Comment:
binding.Marshal now save only exchange, queue and routing key. For headers should also save arguments. Can u implement it?

Reply:
I'll see what I can do

Reply:
This should be taken into account now